karapace/avro_compatibility.py

```diff
@@ -10,7 +10,7 @@
 E = TypeVar("E", bound=Enum)
 
 
-def parse_json_ignore_trailing(s: str) -> Any:
+def parse_avro_schema_definition(s: str) -> Schema:
```

Comment:
renamed both this and `parse_schema_definition` to avoid confusiong, now the names have the format name in it.